subprojects/docs/src/docs/userguide/commandLineInterface.adoc

```diff
@@ -528,6 +528,7 @@ The built-in `gradle wrapper` task generates a script, `gradlew`, that invokes a
 
 You can also specify `--distribution-type=(bin|all)`, `--gradle-distribution-url`, `--gradle-distribution-sha256-sum` in addition to `--gradle-version`. Full details on how to use these options are documented in the <<gradle_wrapper,Gradle wrapper section>>.
 
+[[continuous_build]]
```

Comment:
Heh, I just fixed that broken link in another PR: https://github.com/gradle/gradle/pull/4896

Reply:
Cool. That should be sorted out during my next merge.